mod/config/pkg/spec/special_cases.go

```diff
@@ -20,19 +20,19 @@
 
 package spec
 
-import "math"
-
 // Special cased Bartio for some ad-hoc handling due to the way
 // some bugs were handled on Bartio. To be removed.
 const (
-	BartioChainID uint64 = 80084
+	BartioChainID = TestnetEth1ChainID
```

Comment:
nit: I avoided doing that because I was not sure Bartio was our testnet really. This is fine with me as long as we have no plans to change `TestnetEth1ChainID` to make it point to a different network

Reply:
tbh its not really worth having both "Bartio" and "Testnet" in the code when they refer to the same thing